docker/client.py

```diff
@@ -824,14 +827,21 @@ def logs(self, container, stdout=True, stderr=True, stream=False,
                 params['tail'] = tail
             url = self._url("/containers/{0}/logs".format(container))
             res = self._get(url, params=params, stream=stream)
```

Comment:
Can we reduce the duplication here? This is very similar to the same code in `attach()`.